modes/tmtv/src/index.js

```diff
@@ -151,7 +151,11 @@ function modeFactory({ modeConfiguration }) {
       series: [],
     },
     isValidMode: ({ modalities }) => {
-      const modalities_list = modalities.split('\\');
+      let modalities_list = modalities.split('\\');
+      if (modalities_list.length === 1 && modalities.indexOf('/') !== -1) {
+        // Sometimes the forward slash is used.
```

Comment:
if sometimes forward slash is used, we should just fix it inside the worklist, since now every mode need to implement this fix

Reply:
Sounds good.